modules/ossm-migrating-workloads-using-the-istio-revision-label.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-workloads-using-the-istio-revision-label_{context}"]
+= Migrating workloads using the Istio revision label
+
+Now you can migrate your workloads from the {SMProduct} 2.6 control plane to the {SMproduct} 3.0 control plane.
+
+[NOTE]
```

Comment:
I would probably add another note about revision tags, something like:
Revision tags are not used in this example for simplicity but it's recommended to use them for big meshes to avoid re-labeling of namespaces during future 3.y.z upgrades. See <revision tags doc link>

Reply:
done